### PR TITLE
docs: Add June 2024 newsletter [OR-770]

### PR DESCRIPTION
**Pull request by notlee**
Note: I'll migrate this to the new site when it's approved

### CODE REVIEW

File: apps/astro-website/src/content/posts/2024-07-15-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Our [updated grid guidelines](https://origami-beta.ft.com/guides/grid/) are more comprehensive and, with a smaller number of columns for smaller viewports, more orientated to mobile design than before. We also have an [accurate representation within Figma](<https://www.figma.com/design/xf2Hi9dzeNOAEQdaoqelRy/%F0%9F%92%A0-Grid---Origami-(o3)?node-id=3-108&t=QeTAtPVjhEbhDwC9-0>) design tooling.
+
+![A graphic showing our new grid has 4 columns on mobile and 12 on larger devices.](/assets/images/2024-07-15-newsletter/column-spanning.svg)
+
+We're continuing to gather real world feedback from designers and are working toward a co-ordinated rollout across existing ft.com pages for early 2025.
+
+### Editorial Typography
+
+We've also released new [editorial typography guidelines for o3](https://origami-beta.ft.com/guides/grid/).
```
Comment (lee-campbell-ft) [inline apps/astro-website/src/content/posts/2024-07-15-newsletter.md:33]: Should this link be https://origami-beta.ft.com/guides/typography/?

File: apps/astro-website/src/content/posts/2024-07-15-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+title: June Newsletter
+description: We've released new design guidelines, components, and service dashboards.
+author: Lee Moody
+publishDate: 2024-07-15
+tldr: We released new design guidelines for our grid system and editorial typography; we have new dashboards to maintain the reliability of Origami services; and we're putting together a launch plan for the new beta website, which puts Origami 3 and our new design guidelines front-and-center.
+tags:
+	- Newsletter
+---
+
+## Top things
```
Comment (lee-campbell-ft) [inline apps/astro-website/src/content/posts/2024-07-15-newsletter.md:11]: Should this maybe be a H1 if New Design Guidelines, Editorial Typography etc are sub items?

**Review by lee-campbell-ft — COMMENTED** [review]
Hey, thanks for pulling this together. It looks great! I just made two minor comments; it was interesting to read through, and I learnt some stuff! 

More broadly would be interested to know what the audience/channels are etc. just so I have in my head when thinking about comms etc.

File: apps/astro-website/src/content/posts/2024-07-15-newsletter.md (hunk 2 above)
Comment (notlee) [reply apps/astro-website/src/content/posts/2024-07-15-newsletter.md:11]: Good spot! "June Newsletter" becomes the H1 automatically, so I'll make the sub-sections h3 instead

File: apps/astro-website/src/content/posts/2024-07-15-newsletter.md (hunk 1 above)
Comment (notlee) [reply apps/astro-website/src/content/posts/2024-07-15-newsletter.md:33]: It should!

**Comment by notlee** [comment]
@lee-campbell-ft Thanks for the review! Our monthly newsletter is a broad update from the team. It goes out to tech, brand, product design, and delivery.

**Review by frshwtr — APPROVED** [review]

**Pull request merged**